rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-pipeline-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: pipeline_alerts
+      interval: 1m
+      rules:
+        - alert: HighSchedulingOverhead
+          expr: |
+            (
+                sum by (source_cluster) (increase(pipelinerun_duration_scheduled_seconds_sum{status='succeded'}[30m]))
+                /
+                sum by (source_cluster) (increase(pipelinerun_duration_scheduled_seconds_count{status='succeded'}[30m]))
+            )
+            /
+            (
+                sum by (source_cluster) (increase(tekton_pipelines_controller_pipelinerun_duration_seconds_sum{status='success'}[30m]))
+                /
+                sum by (source_cluster) (increase(tekton_pipelines_controller_pipelinerun_duration_seconds_count{status='success'}[30m]))
+            )  > 0.05
+          for: 12h
+          labels:
+            severity: warning
+            slo: true
+          annotations:
+            summary: >-
+              Tekton has a scheduling overhead vs. PipelineRun durations of >5%
+            description: >-
+              Tekton controller on cluster {{ $labels.source_cluster }} has a ratio of the time needed to receive PipelineRun creation events vs. overall PipelineRun execution time of {{ $value }} instead of less than 0.05.
```

Comment:
Can we break this line for readability (same for the other rule)?

Reply:
For the percentage value, if you want, can also use `{{ $value | humanizePercentage }}` as @kim-tsao did here: https://github.com/redhat-appstudio/o11y/pull/108/files#diff-122adf1739691203cb3c64d8e965e4def93cab660ca866a3456ad9a9c787e3b6R23

Reply:
Sure I'll break up the line, and sort out what `{{ $value | humanizePercentage }} ` provides exactly and curate the message accordingly @yftacherzog thanks